### ISSUE TITLE
Graphical Representation of Polls

### ISSUE THREAD

**Issue by DInnoStudio** [adamosimpson21/wepollclient#2]
Priority: High

Pie Chart or Histogram (dependent on poll type) displayed after a user votes on a question.
If a user has already voted on a poll and they click on the poll again, they should go right to the chart.
Links from chart page to "Analyze" (can be inactive for now) and "Share" (either create a jpg to share, or autoshare on sm sites.  Later this share file will include the user avatar and optionally their vote)
I have a color swatch I've been using for poll answers, attached here. Use the main bar for each color.
![wepollcolorscheme](https://user-images.githubusercontent.com/32209073/47529768-8df4f800-d85d-11e8-80f4-46bedec3b5be.jpg)

**Comment by adamosimpson21** [comment]
Ok, so this is a lot of different issues in one post. I'll try to break them up into smaller parts. 

Creating results charts: 
Difficulty: 4-5. How should I know what questions should be histograms or pie charts? It's pretty easy to flip between the two, there can just be a button for each question that flips between the different views. I'm not sure what types of questions would warrant each type of chart, I feel like users would want to see "part of a whole" pie chart data and "how do I stack up" histogram data for most polls.

User flow directly to results:
Difficulty: 2 . I just have to write some error handling for this. It currently exists, but is not functioning properly

Links from chart page to "Analyze" (can be inactive for now) and "Share" (either create a jpg to share, or autoshare on sm sites. Later this share file will include the user avatar and optionally their vote)
Could we move this to a different issue? I'd like to discuss more what the Analyze feature does and how to build it. I understand the share aspect of it, though we're going to need to flesh out the details of how it works. Having SM share buttons isn't too difficult, but what we're sharing and who can have access to it is a more difficult question. If only users can view results to a poll, are we posting those results to SM? Should we have a 'ghost' user functionality where non-registered users can do everything a user could, but the votes aren't recorded? That's one way to get around unregistered users viewing things. Either way, I'd like to move this chunk to a different area

Color swatches:
Difficulty: 1. Thanks, I needed these. We're going to need a lot more colors though (+20)

**Comment by DInnoStudio** [comment]
I'll pull out the Analyze and Share to 2 different issues.

Creating Results Charts - Pie charts for "choose 1", histogram for "choose multiple".  
I guess creating a "choose multiple" is another issue, I'll create it

**Comment by adamosimpson21** [comment]
It's been a long and grueling day, but I got a bunch of the visualization stuff done. It's not pretty yet, but it's functioning. I'm going to do some styling and a little bit of polish, but I feel like once this feature is ready, we should do some more marketing and show it off to people. It works really well and has a nice 'wow' factor to it, but I'm going to polish it a little to get more of a wow
